frontend/lib/common-steps/ask-nyc-address.tsx

```diff
@@ -59,6 +59,7 @@ function getInitialState(s: AllSessionInfo): OnboardingStep1Input {
   return {
     firstName: "ignore",
     lastName: "ignore",
+    preferredFirstName: "ignore", // not sure if this should change
```

Comment:
Urg I think you can just set this to an empty string.  We really need to just make this form use its own separate mutation, rather than reusing onboarding step 1's mutation.  But since onboarding step 1 has preferred first name as an optional/empty field, this can probably just default to an empty string (it won't make the GraphQL endpoint spit out a validation error).